tests/test_ext_marshmallow_common.py

```diff
@@ -19,13 +19,19 @@ def test_raise_if_schema_class_passed(self):
     def test_same_schemas_instances_equal(self):
         assert make_schema_key(PetSchema()) == make_schema_key(PetSchema())
 
+    def test_same_schemas_instances_with_sets_equal(self):
+        # Note: this test relies on non-deterministic hashing behaviour:
+        for i in range(0, 50):
+            assert make_schema_key(PetSchema(load_only=('1', '2', '3', '4'))) == make_schema_key(PetSchema(load_only=('4', '3', '2', '1')))
+
```

Comment:
Those are not sets but tuples, so I don't think this test tests what it means to.

I understand the reason for the loop, but I wouldn't do that. I'd just test with a set and a loop.

```py
assert make_schema_key(PetSchema(load_only=['1', '2', '3', '4'])) == make_schema_key(PetSchema(load_only=['4', '3', '2', '1']))
assert make_schema_key(PetSchema(load_only=set(['1', '2', '3', '4']))) == make_schema_key(PetSchema(load_only=set(['4', '3', '2', '1'])))
```